src/fairseq2/recipes/wav2vec2/asr/_train.py

```diff
@@ -166,6 +166,22 @@ class Wav2Vec2AsrTrainDatasetSection(DatasetSection):
     npc: int = 10
     """The number of parallel calls to use in the pipeline."""
 
+    # Upsampling
+    beta_corpus: float | None = None
+    beta_language: float | None = None
+    """Params specifying sampling temperature; between [0,1]."""
+
+    # SpecAugment
+    spec_aug_p: float | None = None
+    """Probability of applying SpecAugment per row."""
+    spec_aug_freq_mask_param: int = 80
+    """Maximum frequency mask length."""
+    spec_aug_time_mask_param: int = 80
+    """Maximum time mask length."""
+
+    always_read_tsv: bool = False
+    """If ``True``, always read the TSV manifest, regardless of whether parquet datasets exist."""
```

Comment:
i think this's not clear in the context of fs2 trainer, should not we create our own mms trainer at some point ?